iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -593,6 +594,9 @@ private Path getColStatsPath(Table table) {
   }
 
   private Path getColStatsPath(Table table, long snapshotId) {
+    if (((BaseTable) table).operations() instanceof HadoopTableOperations) {
+      return new Path(table.location() + STATS + snapshotId);
```

Comment:
I am not sure this path is reasonable. Please provide your suggestion here. Thanks.